crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -189,6 +209,10 @@ impl ApiServer {
                 .await
                 .map(|val| AlloyU256::from(val).inner())
                 .to_rpc_result(),
+            // --- Snapshot ---
+            EthRequest::EvmSnapshot(_) => self.snapshot().await.to_rpc_result(),
```

Comment:
```suggestion
            EthRequest::EvmSnapshot(()) => self.snapshot().await.to_rpc_result(),
```

I prefer making the fact that there are no expected params more obvious